Source/WebKit/NetworkProcess/webrtc/NetworkRTCProvider.cpp

```diff
@@ -382,6 +382,13 @@ Ref<NetworkRTCMonitor> NetworkRTCProvider::protectedRTCMonitor()
     return m_rtcMonitor;
 }
 
+#if PLATFORM(COCOA)
```

Comment:
I think we should just move this function to the existing `#if PLATFORM(COCOA)` block earlier in this file.

Reply:
sure, we can do that.